src/framework/app-base.js

```diff
@@ -1794,15 +1794,16 @@ class AppBase extends EventHandler {
      * @param {boolean} [depthTest] - Specifies if the sphere lines are depth tested against the
      * depth buffer. Defaults to true.
      * @param {Layer} [layer] - The layer to render the sphere into. Defaults to {@link LAYERID_IMMEDIATE}.
+     * @param {Mat4} [mat] - Matrix to transform the box before rendering.
      * @example
      * // Render a red wire aligned box
      * const min = new pc.Vec3(-1, -1, -1);
      * const max = new pc.Vec3(1, 1, 1);
      * app.drawWireAlignedBox(min, max, pc.Color.RED);
      * @ignore
      */
-    drawWireAlignedBox(minPoint, maxPoint, color = Color.WHITE, depthTest = true, layer = this.scene.defaultDrawLayer) {
-        this.scene.immediate.drawWireAlignedBox(minPoint, maxPoint, color, depthTest, layer);
+    drawWireAlignedBox(minPoint, maxPoint, color = Color.WHITE, depthTest = true, layer = this.scene.defaultDrawLayer, mat = undefined) {
```

Comment:
Why do you need the `= undefined` here?

Reply:
Because I'm still used to c++ parameter rules I guess?! Removed, thanks.